### ISSUE TITLE
[BUG] Another Person registered using my phone number without OTP

### ISSUE THREAD

**Issue by siddarthahs123** [egovernments/DIVOC#576]
**Describe the bug**

I try to register for Vaccination using my phone number, but after entering the OTP in CoWIN/Arogya Setu App, the page is showing up details of an another person who had already received the first dose of vaccination with certificate. I am still bewildered as to how other person was able to register without the OTP.

My phone num : 8867265618

**To Reproduce**
1. Go to 'https://www.cowin.gov.in/home'
2. Click on 'Click Register/Sign In yourself'
3. Scroll down to 'Enter your phone number and enter the OTP which you have received to register for Vaccination'

**Expected behavior**

Without OTP other person should not register using my phone number

**Screenshots**
![image](https://user-images.githubusercontent.com/15208583/117940916-35a47980-b327-11eb-9f70-3a4d901d27c6.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
Please fix this issue ASAP

**Comment by PRG8190** [comment]
According to me this is not bug. Vaccine center registered wrong number for that person you have mentioned in your photo. Earlier vaccine centers was taking walk in enteries. My Dad took his 1st dose of vaccine ( Walk in) but it's not reflecting in coWin so I am assuming they entered wrong number may be 1 different number from 10 numbers. Because of there mistake this thing is happening and they don't have any solution.

**Comment by siddarthahs123** [comment]
Hi,

Thank you for responding.

Since some other person already vaccinated using my phone number, how should I register for my vaccine?

I am not ok to be add myself as a beneficiary for some random person.

Please look this as a high priority and help me with this

Thanks

**Comment by PRG8190** [comment]
This is actually wrong number registeration issue.. Clerical mistake of Vaccine centers.. So many people are complaining this issue with different subject.. There is no option given in coWin app for editing wrong name number age etc. You can use your alternative number for your registration.. coWin should give access for login with Beneficiary reference ID provided by vaccine centers after 1st dose which makes things more easier..

**Comment by siddarthahs123** [comment]
Hi,

I don't have any alternative number. Everything is linked to this number. I can't buy new sim now because of the lockdown.

**Comment by PRG8190** [comment]
You can register 4 persons with one mobile number.. linked number is not necessary

**Comment by siddarthahs123** [comment]
I will consider this conversation as a confirmation and will register for my vaccine as a beneficiary.

Hope this will not impact anything in future

Thanks for your help

**Comment by PRG8190** [comment]
No.. You cannot trust these govt website for future.. I also have a issue which is not solved yet.. So don't consider anything.. Yes you can register yourself with different number which is mentioned in govt advisory..
